internal/workload/podspec_updates_test.go

```diff
@@ -911,6 +915,91 @@ func TestPodTemplateAnnotations(t *testing.T) {
 
 }
 
+func TestQuitUrlEnvVar(t *testing.T) {
+
+	var (
+		u = workload.NewUpdater("cloud-sql-proxy-operator/dev", workload.DefaultProxyImage)
+	)
+
+	// Create a pod
+	wl := podWorkload()
+	wl.Pod.Spec.Containers[0].Ports =
+		[]corev1.ContainerPort{{Name: "http", ContainerPort: 8080}}
+
+	// Create a AuthProxyWorkload that matches the deployment
+	csqls := []*cloudsqlapi.AuthProxyWorkload{
+		simpleAuthProxy("instance1", "project:server:db"),
+		simpleAuthProxy("instance2", "project:server2:db2"),
+		simpleAuthProxy("instance3", "project:server3:db3")}
+
+	csqls[0].ObjectMeta.Generation = 1
+	csqls[1].ObjectMeta.Generation = 2
+	csqls[2].ObjectMeta.Generation = 3
+
+	var wantQuitUrlsEnv = fmt.Sprintf(
+		"http://localhost:%d/quitquitquit http://localhost:%d/quitquitquit http://localhost:%d/quitquitquit", workload.DefaultAdminPort,
```

Comment:
nit: shorter line length is easier to read. how about this?

``` golang
var wantQuitUrlsEnv = strings.Join(
  []string{
    fmt.Sprinf("http://localhost:%d/quitquitquit", workload.DefaultAdminPort),
    fmt.Sprinf("http://localhost:%d/quitquitquit", workload.DefaultAdminPort),
    fmt.Sprinf("http://localhost:%d/quitquitquit", workload.DefaultAdminPort),
  },
  " ",
)

Reply:
Fixed.